crates/ruff/src/rules/flake8_simplify/snapshots/ruff__rules__flake8_simplify__tests__SIM222_SIM222.py.snap

```diff
@@ -116,4 +116,242 @@ SIM222.py:30:4: SIM222 [*] Use `True` instead of `... or True`
 32 32 | 
 33 33 | 
 
+SIM222.py:47:4: SIM222 [*] Use `True` instead of `... or True`
+   |
+47 | if a or "" or True:  # SIM222
+   |    ^^^^^^^^^^^^^^^ SIM222
+48 |     pass
+   |
+   = help: Replace with `True`
+
+ℹ Suggested fix
+44 44 |     pass
+45 45 | 
+46 46 | 
+47    |-if a or "" or True:  # SIM222
+   47 |+if True:  # SIM222
+48 48 |     pass
+49 49 | 
+50 50 | if a or "foo" or True or "bar":  # SIM222
+
+SIM222.py:50:14: SIM222 [*] Use `True` instead of `... or True`
+   |
+50 |     pass
+51 | 
+52 | if a or "foo" or True or "bar":  # SIM222
+   |              ^^^^^^^^^^^^^^^^^ SIM222
+53 |     pass
+   |
+   = help: Replace with `True`
+
+ℹ Suggested fix
+47 47 | if a or "" or True:  # SIM222
+48 48 |     pass
+49 49 | 
+50    |-if a or "foo" or True or "bar":  # SIM222
+   50 |+if a or "foo":  # SIM222
+51 51 |     pass
+52 52 | 
+53 53 | if a or 0 or True:  # SIM222
+
+SIM222.py:53:4: SIM222 [*] Use `True` instead of `... or True`
```

Comment:
Can we update these diagnostic messages, since they no longer reflect the suggestion IIUC?

Reply:
Done!